### ISSUE TITLE
False positive? SVG title element + aria-labelledby gives error message.

### ISSUE THREAD

**Issue by sarahbrodwall** [ThePacielloGroup/ARC-Toolkit#24]
**Describe the bug**

When I attempt to give an SVG an accessible name by using `aria-labelledby` to refer to the SVG's `title` element, ARC Toolkit returns an error "Active SVG missing `aria-label`".  The Name computation pane in Chrome's DevTools shows the accessible name computation working as expected.  Narrator, JAWS and NVDA identify the element as expected.  

**To Reproduce**

1. Go to [this test page](https://sarahbrodwall.github.io/tests/svg-aria-labelledby.html).
2. Open ARC Toolkit and click on "Run tests".
4. Receive error result: "1  Error(s)Active SVG missing `aria-label` (activeSvgMissingAriaLabel)
Description: The <svg> element is the sole content of an actionable element (such as a link or button) but neither the <svg> nor the actionable element has an accessible name."

**Expected behavior**

I would not expect to receive an error in this case.

**Version information**

- Browser and version: Chrome on Windows 10, version 107.0.5304.107 (Official Build) (64-bit)
- ARC Toolkit version: 5.4.2

**Comment by poorgeek** [comment]
Thanks @sarahbrodwall, we're looking into this.

**Comment by jducrot** [comment]
This is still an issue. The aria-labelledby in the screenshot point sto visible text and the button shows a valid accessible name in the accessibility pane of the console. Additionally the inspect element button doesn't work for any of these elements. The problem is currently visible on the homepage of https://www.whitehouse.gov/

![tpgi-arc-svg-button](https://github.com/ThePacielloGroup/ARC-Toolkit/assets/280674/86fd0a5a-2cf0-4688-966c-ddaf7892d5e4)

![tpgi-arc-svg-button-2](https://github.com/ThePacielloGroup/ARC-Toolkit/assets/280674/fad2509b-138f-4fff-872e-3a597e44abc6)

**Comment by ferllings** [comment]
Internal refs:
- https://dev.azure.com/tpgarc/AAP%20Portal%20R2/_workitems/edit/8481
- https://github.com/ThePacielloGroup/ARC-RulesEngine/pull/414

**Comment by poorgeek** [comment]
Closing as this issue was fixed in 5.6.2
